packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -50,6 +50,10 @@ export function initializeResponsiveMode(element?: HTMLElement): void {
   }
 }
 
+export const getInitialResponsiveMode = () => {
+  return _defaultMode || _lastMode || ResponsiveMode.large;
+};
+
```

Comment:
Small thing but can you switch this to `function` syntax to match the rest of the file? 
```suggestion
export function getInitialResponsiveMode() {
  return _defaultMode || _lastMode || ResponsiveMode.large;
}

```

I still think `function` is preferable to lambdas in general for exported utilities since it makes debugging easier (IIRC lambdas show up as anonymous functions in callstacks, whereas functions show up with names), but that's probably something to discuss separately.